crates/driver/src/node/mod.rs

```diff
@@ -0,0 +1,94 @@
+use {
+    crate::logic::eth,
+    ethcontract::{transport::DynTransport, Web3},
+    thiserror::Error,
+};
+
+pub mod contracts;
+
+const MAX_BATCH_SIZE: usize = 100;
+
+#[derive(Debug, Error)]
+pub enum Error {
+    #[error("method error: {0:?}")]
+    Method(#[from] ethcontract::errors::MethodError),
+    #[error("deploy error: {0:?}")]
+    Deploy(#[from] ethcontract::errors::DeployError),
+}
+
+/// The Ethereum node.
+#[derive(Debug)]
+pub struct EthNode(Web3<DynTransport>);
```

Comment:
I'm not super fond of this, as to me this doesn't represent an "Ethereum node" but more "Blockchain state".

Reply:
I would call it `Chain` or `Blockchain` if anything - just so we are more chain agnostic in our naming.

`Eth` is fine as well.